### PR TITLE
Reverse a stack

### PR DESCRIPTION
**Pull request by ishwari20**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Out of the two given stacks, one is populated and the other one is empty. Write a function that reverses the populated stack using the one which is empty.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/other/Find the Unique Element in array.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/*
```
Comment (gat786) [inline C-Plus-Plus/other/Find the Unique Element in array.cpp:1]: This file should not be a part of this pr. Maybe you should try and remove it.

File: C-Plus-Plus/stack/Reverse_a_Stack.cpp [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/* 
+Reverse a Stack
+Out of the two given stacks, one is populated and the other one is empty. Write a function that reverses the populated stack using the one which is empty.
+*/
+
+#include <iostream>
+#include <stack>
+#include <stack>
```
Comment (gat786) [inline C-Plus-Plus/stack/Reverse_a_Stack.cpp:8]: why is stack imported twice?

File: C-Plus-Plus/stack/Reverse_a_Stack.cpp [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+
+/*
+Sample Input 1:
+6
+1 2 3 4 5 10
+Sample Output 1:
+1 2 3 4 5 10
+*/
```
Comment (gat786) [inline C-Plus-Plus/stack/Reverse_a_Stack.cpp:63]: add the complexities here

**Review by gat786 — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
@ishwari20 Don't push more than one code file in a single PR.

**Pull request closed without merging**